### DISCUSSION TITLE
Terminate Quick Support if admin rights not given

### DISCUSSION THREAD

**Discussion by bilogic** [rustdesk/rustdesk#7882] [Q&A] [answered]
## Problem

rustdesk runs even if user clicks `No` at the `UAC`

We have an admin software to support, and we get stuck in a bind here whenever the user clicks `No` at the `UAC`.
To the user, rustdesk app shows up, so everything must be ok. But when we connect, we can't control the admin software.

The amount of effort we have to spend getting the user to click on `Yes` is crazy. An automated solution will really help!

## Actual

On `1.2.3-2-x86`,

1. From https://github.com/rustdesk/rustdesk/discussions/5618, I renamed my `.exe` to end with `-qs.exe` so that it requests for admin via `UAC`
2. However, rustdesk runs even if user clicks `No` at the `UAC`

## Expectation/Solution
1. Is there an option to stop rustdesk from running if the user clicks `No` at `UAC`? 
2. If not, can I propose `-qas.exe`?

Btw, adding `-qs-` in the filename has no effect, rustdesk just runs, the `UAC` prompt does not show.

# AFTER ANSWER UPDATE

I don't see how #8981 addresses this problem...

**Comment by Somnius** [comment]
Maybe make a launcher file that elevates perms and if not cancel the launching process and use that as desktop and start menu shortcut

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/discussions/8981#discussioncomment-10263847
